02/app.js

```diff
@@ -1,11 +1,24 @@
 const calendarJS = {
-    'ES1': '1997-06',
-    'ES2': '1998-06',
-    'ES3': '1999-12',
-    'ES4': null,
-    'ES5': '2009-12',
-    'ES6': '2015-06',
-    'ES7': '2016-06',
-    'ES8': '2017-06',
-    'ES9': '2018-06',
-}
\ No newline at end of file
+	ES1: "1997-06",
+	ES2: "1998-06",
+	ES3: "1999-12",
+	ES4: null,
+	ES5: "2009-12",
+	ES6: "2015-06",
+	ES7: "2016-06",
+	ES8: "2017-06",
+	ES9: "2018-06",
+};
+
+for (const key in calendarJS) {
+	const keyValue = calendarJS[key];
+	if (keyValue !== null) {
+		console.log(key + " wydano w terminie " + keyValue);
+	} else {
+		console.log(key + " nie zostało wydane");
+	}
+}
+
+if (typeof this[isbn]["title"]["pl"] === "undefined") {
+	return null;
+}
```

Comment:
Dlaczego tu tutaj jest? :P

Reply:
W sumie nie wiem ;) nie jest to tam potrzebne właśnie widzę ;)